providers/redfish/firmware.go

```diff
@@ -138,6 +161,47 @@ func (c *Conn) FirmwareInstallStatus(ctx context.Context, installVersion, compon
 	switch {
 	case strings.Contains(vendor, constants.Dell):
 		task, err = c.dellJobAsRedfishTask(taskID)
+	case strings.Contains(vendor, constants.Packet):
+		resp, _ := c.redfishwrapper.Get("/redfish/v1/TaskService/Tasks/" + taskID)
```

Comment:
You might be able to use the TaskService method from Gofish  - https://github.com/stmcginnis/gofish/blob/6e8a2909577b8424e751890a5c432352f227097b/serviceroot.go#L257